config.example/playbooks/example.yml

```diff
@@ -0,0 +1,14 @@
+---
+# Your custom playbooks should go in your DeepOps configuration directory,
+# under `config/playbooks`.
+# These playbooks can be used to make any customizations to your cluster
+# that aren't already provided by DeepOps!
+# For example, this playbook installs `cowsay` on all your Kubernetes nodes:
+
+- hosts: kube-node
+  become: yes
+  tasks:
+  - name: install cowsay
```

Comment:
+1 for cowsay. Let's hope nobody runs this locally and can't figure out how to turn it off. :)

Reply:
They can learn about `state: absent` ;-)